docs/releases/4.2.txt

```diff
@@ -135,6 +135,11 @@ Minor features
 
 * ...
 
+Database backends
```

Comment:
```suggestion
This should be in the `Models` section.
```

Reply:
Thank you for suggestion, I am going add it in next improvement